nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -2047,6 +2047,34 @@ Superuser created successfully.
           <literal>java-packages.compiler</literal>.
         </para>
       </listitem>
+      <listitem>
```

Comment:
This needs to move to a prominent location in rl-2205

Reply:
I have moved the relevant release notes to rl-2205.